.github/workflows/build_validation_workflow.yml

```diff
@@ -2,18 +2,76 @@ name: Build Validation Workflow
 
 on:
   workflow_call:
+    inputs:
+      image_name:
+        type: string
+        description: "Execution Environment"
+        required: false
+      model_type:
+        type: string
+        description: "type of model to execute"
+        required: false
+      is_docker:
+        type: boolean
+        description: "Is Docker used for build validation?"
+        required: false
+        default: false
 
 env:
   PYTHON_VERSION: "3.9"
 
 jobs:
+  run-unit-tests-docker:
+    if: ${{ inputs.is_docker == true }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+
+      - name: Load Python Dependencies
+        shell: bash
+        run: |
+          set -e
+          python -m pip install --upgrade pip
+          python -m pip install --upgrade -r .github/requirements/build_validation_requirements.txt      
+      - name: Lint Code
+        run: flake8
+
+      - name: Build Docker container
+        shell: bash
+        run: |
+          echo "Building Docker container for validation with tag: ${{ inputs.image_name }}"
+          docker build -t ${{ inputs.image_name }} -f mlops/${{ inputs.model_type }}/environment/Dockerfile mlops/${{ inputs.model_type }}/environment
```

Comment:
This appears to make having a Dockerfile a hard requirement.  What happens if a model doesn't include a Dockerfile?  If we do want a Dockerfile to be a requirement, we should add it to the docs.

Reply:
I updated the docs. Thank you